ocs_ci/ocs/constants.py

```diff
@@ -75,6 +75,7 @@
     TEMPLATE_CNV_VM_WORKLOAD_DIR, "vm-standalone-pvc"
 )
 DATA_DIR = os.getenv("OCSCI_DATA_DIR") or os.path.join(TOP_DIR, "data")
+BIN_DIR = os.path.join(TOP_DIR, "bin")
```

Comment:
nope, bin dir is not from TOP_DIR directory, we already have bin dir define in config, so it should be directly used from config value

Reply:
done